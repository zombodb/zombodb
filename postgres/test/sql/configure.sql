CREATE OR REPLACE FUNCTION assert(actual int8, expected int8, message text) RETURNS bool LANGUAGE plpgsql AS $$
BEGIN
  IF expected IS DISTINCT FROM actual THEN
    RAISE EXCEPTION 'ASSERT: %: expected=%, actual=%', message, expected, actual;
  END IF;

  RETURN true;
END;
$$;
