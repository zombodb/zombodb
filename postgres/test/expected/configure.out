CREATE OR REPLACE FUNCTION assert(actual int8, expected int8, message text) RETURNS bool LANGUAGE plpgsql AS $$
BEGIN
  IF expected IS DISTINCT FROM actual THEN
    RAISE EXCEPTION 'ASSERT: %: expected=%, actual=%', message, expected, actual;
  END IF;

  RETURN true;
END;
$$;
ALTER TABLE so_posts SET (autovacuum_enabled = false);
ALTER TABLE so_users SET (autovacuum_enabled = false);
ALTER TABLE words SET (autovacuum_enabled = false);
